ext/blake2_ext/blake2-impl.h

```diff
@@ -1,20 +1,23 @@
 /*
```

Comment:
> keeping them vendored in your gem though since that allows you to verify that you have the correct files in place and that they have not been modified.

I wish I could provide the guarantee **to the users** though. Whatever is here on GitHub could be different from what's on Rubygems. But yeah, let's keep the source here.

> If you just pull them in blindly when you build you risk a MITM on those files being modified (or not available any longer, like when they recently re-hosted them to Github).

Yep that's the other subtility I want to address. I could be malicious or imprudent, the result would be the same. I'll figure it out. Thanks!